app/views/layouts/application.html.slim

```diff
@@ -19,6 +19,7 @@ html
         = content_tag :div, msg, class: build_alert_classes(type)
     
   .container      
-    = yield
-  footer              
-    == render 'shared/footer'
\ No newline at end of file
+    = yield  
+    - unless action_name == "show" && controller_name == "cars"  
+      footer      
+        == render 'shared/footer'
```

Comment:
Wait, why don't we want to show the footer when we show a car?

Reply:
Because somehow it's covering the pagination and don't want to mess there with all that JS and turbo streams :) But if I must I'll try to figure something, maybe can move it to the right

Reply:
No, it's a ruby bootcamp, we should focus on rails and meddle with other languages only if we absolutely need them